checks/containers/sarus/check_pyfr.py

```diff
@@ -1,3 +1,8 @@
+# Copyright 2016-2023 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
```

Comment:
add 
```
prerun_cmds += ['unset XDG_RUNTIME_DIR']
```
or
```
self.env_vars['XDG_RUNTIME_DIR'] = ''
```

Reply:
It's done in the setup container platform